src/components/QuickOpenModal.js

```diff
@@ -68,6 +68,10 @@ function filter(values, query) {
   });
 }
 
+const styles = {
```

Comment:
Style in the CSS file.

Reply:
for some reason when I try it this way I can't seem to get it to continue highlighting. I imported the css file, and tried to attach className='highlight'.

I may just be messing up the syntax?